tools/VscPreviewExtension/README.md

```diff
@@ -30,7 +30,7 @@ For further information and details about DocFX Flavored Markdown, please refere
 | `ctrl+k q` | `Open Dfm Preview to the side` | Preview to side |
 |  | `show Dfm Show` | Show Source |
 
-  ![PreviewToside](/tools/VscPreviewExtension/images/previewToSide.gif)
+  ![PreviewToside](https://github.com/dotnet/docfx/raw/master/tools/VscPreviewExtension/images/previewToSide.gif)
```

Comment:
cause it will be translated to absolute path related to docfx repo root path 

---
In reply to: [126849285](https://github.com/dotnet/docfx/pull/1857#discussion_r126849285) [](ancestors = 126849285)